src/models/ITestType.ts

```diff
@@ -45,8 +45,9 @@ export enum ForVehicleType {
   Hgv = "hgv",
   Psv = "psv",
   Trl = "trl",
-  Car = "car",
   Lgv = "lgv",
+  Car = "car",
+  SmallTrl = "small trailer",
```

Comment:
```suggestion
  SmallTrl = "small trl",
```
should this not be `small trl`?

Reply:
That is right, corrected.